components/combobox/stories/combobox.mdx

```diff
@@ -0,0 +1,89 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as ComboboxStories from './combobox.stories';
+
+<Meta of={ComboboxStories} title="Docs" />
+
+<Title of={ComboboxStories} />
+<Description of={ComboboxStories} />
+
+## Usage notes
+### General notes
+- Combobox uses `.spectrum-PickerButton` instead of a `.spectrum-Picker`
+- The following classes must be added:
+  - `.spectrum-Combobox-textfield` is required on the Textfield outer element (`.spectrum-Textfield`)
+  - `.spectrum-Combobox-input` is required on the `<input>` element inside of Textfields (`.spectrum-Textfield-input`)
+  - `.spectrum-Combobox-button` is required on the FieldButton (`.spectrum-ActionButton spectrum-ActionButton--sizeM`)
+
+### Component separated from InputGroup
+This component was previously a variant style for InputGroup. **InputGroup is deprecated**. The classes listed below containing `InputGroup` have been renamed or removed:
+- `.spectrum-InputGroup-textfield` -> `.spectrum-Combobox-textfield`
+- `.spectrum-InputGroup-input` -> `.spectrum-Combobox-input`
+- `.spectrum-InputGroup-button` -> `.spectrum-Combobox-button`
+- `.InputGroup` -> removed
```

Comment:
Would this be better under a "Migration guide" heading 2? Same goes for this section of the DatePicker docs.

I'm still thinking about a better place for this info to live. What if we moved it to the component changelog for version 1.0?

Reply:
Yeah, this is a good point. This info doesn't really qualify as "usage notes." I also wondered if this info was still relevant to include.

The changelogs are automatically generated, yes? I wonder if I can add something to an older version without it being overridden. I would think so since it just adds new info to the top